starter_kits/TypeScript/hlt/gameMap.ts

```diff
@@ -0,0 +1,141 @@
+import { Ship } from "./Ship";
+import { Direction } from "./Direction";
+import { Position } from "./Position";
+import { Constants } from "./Constants";
+import { MapCell } from "./MapCell";
+import { Entity } from "./Entity";
+
+/**
+ * The game map.
+ *
+ * Can be indexed by a position, or by a contained entity.
+ * Coordinates start at 0. Coordinates are normalized for you.
+ */
+export class GameMap {
+    readonly cells: MapCell[][];
+    public width: number = 0;
```

Comment:
Not a big issue, but defining `= 0` will implicitly set `number` type. So we could delete `:number` everywhere.

Reply:
Removed everywhere.